coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = result.__dict__
```

Comment:
Also I don't why gitmate-bot is giving error on quote marks still :sweat_smile: If you see my forked repo, then you can see that I have made the required changes :smile: